File: HelpSoftware/Help/Help.Web/Web/src/app/EF/AppError.ts
export class AppError {
  ID_AppError: Number;
  ErrorMessage: string;
  StrackTrace: string;
  Created: Date;

  deserialize(input) {
    this.ID_AppError = input.ID_AppError;
    this.ErrorMessage = input.ErrorMessage;
    this.StrackTrace = input.StrackTrace;
    this.Created = input.Created;
    return this;
  }
}

File: HelpSoftware/Help/Help.EF/Entitys/AppError.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Help.EF
{
  public class AppError : IHelpEntity
  {
    [Key]
    public virtual int ID_AppError { get; set; }

    public virtual string ErrorMessage { get; set; }

    public virtual string StackTrace { get; set; }

    public virtual DateTime Created { get; set; }

    [Timestamp]
    public virtual byte[ ] TimeStamp { get; set; }

    public string GetEntityName()
    {
      return "AppError";
    }

    public int GetID()
    {
      return ID_AppError;
    }

    public bool GetWriteHistoryEntry()
    {
      return false;
    }
  }
}